tools/subgraph-example/package.json

```diff
@@ -1,20 +1,49 @@
 {
-  "name": "@hedera-subgraph-greeter-example",
-  "license": "Apache-2.0",
+  "name": "hardhat-project",
+  "devDependencies": {
```

Comment:
Q: lots of new `devDependencies`.
Just making sure but do we need all them or was this added by a dependency package

Reply:
Most of them are auto-generated by hardhat